src/EditorFeatures/Core.Wpf/InlineDiagnostics/AbstractDiagnosticsTaggerProvider.SingleDiagnosticKindPullTaggerProvider.cs

```diff
@@ -38,14 +39,12 @@ private sealed class SingleDiagnosticKindPullTaggerProvider(
         AbstractDiagnosticsTaggerProvider<TTag> callback,
         DiagnosticKind diagnosticKind,
         IThreadingContext threadingContext,
-        IDiagnosticService diagnosticService,
         IDiagnosticAnalyzerService analyzerService,
```

Comment:
note, the IDiagnosticAnalyzerService is where we actually get the diagnostics from.  the IDiagnosticService is the legacy service that serves to push out notifications that diagnostics cahnged to registered listeners.  We're removing hte latter.